datadome/provider.go

```diff
@@ -8,15 +8,16 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+// Provider of DataDome
 func Provider() *schema.Provider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
-			"host": &schema.Schema{
+			"host": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("DATADOME_HOST", nil),
 			},
-			"apikey": &schema.Schema{
+			"apikey": {
```

Comment:
IDK which api key we are talking here, maybe it is worth to be more specific in the naming?

Reply:
If we change the naming of the field, it will imply a breaking change.

Maybe can we add a better description of this field in the `docs/index.md`. WDYT?

Reply:
I have updated the documentation on [65b2631](https://github.com/DataDome/terraform-provider-datadome/pull/17/commits/65b26311d963493f4203169282aec596c07def6e)